src/createServer.js

```diff
@@ -1,11 +1,37 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+const {
+  getUserById, getAllUsers, createNewUser, removeUser, updateUser,
+} = require('./users');
+const {
+  getAllExpenses, getExpense, createNewExpense, removeExpense, updateExpense,
+} = require('./expenses');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express().use(cors());
+
+  app.users = [];
+  app.expenses = [];
```

Comment:
better to create 2 separate variables
```suggestion
  let users = [];
  let expenses = [];
```